scripts/setup_plugin_repos.py

```
@@ -1,151 +1,84 @@
+
     PLUGIN_REPOS_DIR.mkdir(exist_ok=True)
-    
+
     created = 0
     skipped = 0
-    errors = 0
-    
-    print(f"Setting up plugin repository links...")
-    print(f"  Source: {GITHUB_DIR}")
+
+    print(f"Setting up plugin symlinks...")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)